src/Symfony/Component/Validator/Constraints/MacAddress.php

```diff
@@ -28,25 +27,16 @@ class MacAddress extends Constraint
         self::INVALID_MAC_ERROR => 'INVALID_MAC_ERROR',
     ];
 
-    public string $message = 'This is not a valid MAC address.';
-
-    /** @var callable|null */
-    public $normalizer;
+    public ?\Closure $normalizer;
 
     public function __construct(
-        array $options = null,
-        string $message = null,
+        public string $message = 'This is not a valid MAC address.',
```

Comment:
let's also fix the message : "This value is not", related to #53420

Reply:
done

Reply:
I didn't update the translation files as I think you will be doing that in #53420 and the changes will eventually land in the `7.1`  anyway.